openslides_backend/models/motion_change_recommendation.py

```diff
@@ -0,0 +1,35 @@
+from ..shared.patterns import Collection
+from . import fields
+from .base import Model
+
+
+class MotionChangeRecommendation(Model):
+    """
+    Model for motion change_recommendation.
+
+    Reverse fields:
+    - motion_ids
+    """
+
+    collection = Collection("motion_change_recommendation")
+    verbose_name = "motion_change_recommendation"
+
+    id = fields.IdField(description="The id of this motion change recommendation.")
+    title = fields.RequiredCharField(
+        description="The title of this motion change recommendation."
+    )
+    type = fields.PositiveIntegerField("The type of this motion change recommendation.")
+    other_description = fields.CharField(
+        "The other description of this motion change recommendation."
+    )
+    line_from = fields.PositiveIntegerField(
+        "The line from of this motion change recommendation."
+    )
+    line_to = fields.PositiveIntegerField(
+        "The line to of this motion change recommendation."
+    )
+    # TODO add the other Foreign Keys.
+    # TODO rejected: boolean;
+    # TODO internal: boolean;
+    # TODO text: HTML;
+    # TODO creation_time: datetime;
```

Comment:
Boolean fields and Timestamp fields (datetime) implemented in #101. HTML field is a text field for now, maybe we'll add a special field there later for validation.

Reply:
Merged #101 and added the fields.